### ISSUE TITLE
Upgrade to Solr 3.1, required for search issues

### ISSUE THREAD

**Issue by patrickzurek** [eXtensibleCatalog/Drupal-Toolkit#333]
JIRA issue created by: rcook
Originally opened: 2011-05-16 06:56 PM

Issue body:
 (nt)

**Comment by patrickzurek** [comment]
JIRA Comment by user: rcook
JIRA Timestamp: 2011-05-16 07:01 PM

Comment body:

Discussed in team meeting today that we should proceed with the upgrade from our current 1.4.1 to 3.1.   Per testing done by Peter, this is required in order to support both decomposed and non-decomposed UTF8.  All of Rochester's data is 3 bit based Unicode and requires this upgrade to fully support correct searching.

From an email from Peter. 

I have investigated the possibile solutions of the accent problem. I have tested 3 different approaches, and finally it came to light, that only one method is fit for our data. This approace (ICU filter) requires newest Solr (version 3.1), so as a side effect, I have to study the difference between our current 1.4.1 and the 3.1 version (because merging the 2.x version of Lucene into 3.x version of both Solr and Lucene, there is no 2.x version of Solr). Finally I setup a test Drupal server with test data I received from Ben.
